frontend/src/components/file-explorer/FileExplorer.tsx

```diff
@@ -243,6 +274,27 @@ function FileExplorer({ error, isOpen, onToggle }: FileExplorerProps) {
               <p className="text-neutral-300 text-sm">{error}</p>
             </div>
           )}
+          {isOpen && (
+            <button
```

Comment:
Since this is opening a new window, we should probably make it an `a href` and just override the click behavior. Optional though

Reply:
I'll probably hold off a bit here and leave this for future improvement now -- seems `disabled` is not supported for `a href` (and I'm afraid to break things in the last minute 😓 )